hq/test/aws/ec2/EC2Test.scala

```diff
@@ -140,6 +141,22 @@ class EC2Test extends FreeSpec with Matchers with Checkers with PropertyChecks w
         sortedResult should be(sortedResult.sortWith { case ((_, s1), (_, s2)) => s1.size > s2.size })
       }
     }
+
+    "add cloudformation stackId & stackName" in {
+      val sgs1 = new SecurityGroup().withGroupId("id-1").withTags(new Tag().withKey("aws:cloudformation:stack-id").withValue("stack-id-1"), new Tag().withKey("aws:cloudformation:stack-name").withValue("stack-name-1"))
+      val sgs2 = new SecurityGroup().withGroupId("id-2").withTags(new Tag().withKey("aws:cloudformation:stack-id").withValue("stack-id-2"), new Tag().withKey("aws:cloudformation:stack-name").withValue("stack-name-2"))
+      val sgs3 = new SecurityGroup().withGroupId("id-3")
+      val result = new DescribeSecurityGroupsResult().withSecurityGroups(sgs1, sgs2, sgs3)
+      val sgsOpenPort1 = SGOpenPortsDetail("warning", "eu-west-1", "name-1", "id-1", "vpc-1", "tcp", "8080", "Red", false )
+      val sgsOpenPort2 = SGOpenPortsDetail("warning", "eu-west-2", "name-2", "id-2", "vpc-2", "tcp", "8080", "Red", false )
+      val sgsOpenPort3 = SGOpenPortsDetail("warning", "eu-west-1", "name-4", "id-4", "vpc-4", "tcp", "8080", "Red", false )
```

Comment:
Minor: Is there a reason that `sgsOpenPort3` is using `4`?   `"name-4", "id-4", "vpc-4"` would it make more sense to use `3`?